mizar/dp/mizar/workflows/builtins/networkpolicies/triggers.py

```diff
@@ -34,7 +34,7 @@
 @kopf.on.create('networking.k8s.io', 'v1', 'networkpolicies', retries=OBJ_DEFAULTS.kopf_max_retries, when=LAMBDAS.k8s_provider_vanilla)
 async def builtins_on_networkpolicy(body, spec, **kwargs):
     param = HandlerParam()
-    param.name = "{}:{}".format(kwargs['namespace'],kwargs['name'])
```

Comment:
Why are you changing param.name?

Reply:
namespace should be part of body: param.body['metadata']

https://github.com/CentaurusInfra/mizar/blob/7112bc78c8e9d2a175bbe3067793d3258da179cf/mizar/dp/mizar/workflows/builtins/pods/create.py#L58

Reply:
I just add a space here when I notice the tiny format issue. The code was working.

Reply:
I am actually asking you to change to param.name = kwargs['name']
And if you want get the namespace of the network policy, you can use param.body['metadata']['namespace']